doc/examples/bundle_extraction.py

```diff
@@ -153,10 +153,12 @@
 
 ren = window.Renderer()
 ren.SetBackground(1, 1, 1)
-ren.add(actor.line(model_bundle, colors=(1,0,1)))
-ren.add(actor.line(recognized_bundle, colors=(1,1,0)))
+ren.add(actor.line(model_bundle, colors=(.1,.7,.26)))
+ren.add(actor.line(recognized_bundle, colors=(.1,.1,6)))
 window.record(ren, out_path='CST_L_recognized_bundle.png',
-              size=(600, 600))
```

Comment:
I believe these lines should be: 

```
ren.set_camera(focal_point=(-18.17281532, -19.55606842, 6.92485857),
               position=(-360.11, -340.46, -40.44),
               view_up=(-0.03, 0.028, 0.89))

window.record(ren, out_path='CST_L_recognized_bundle.png', size=(600, 600))
```

Reply:
Hello Ariel,

Thanks for point out the typo. I have fixed it and also camera view for AF_L bundle.

here's how they look now:
![af_l_recognized_bundle](https://user-images.githubusercontent.com/26060471/49691107-6c985480-fb09-11e8-8d19-1ec6b6f20215.png)

![cst_l_recognized_bundle](https://user-images.githubusercontent.com/26060471/49691109-728e3580-fb09-11e8-8878-52e309436395.png)

Reply:
@Garyfallidis